js/index.js

```diff
@@ -0,0 +1,130 @@
+function showMenu() {
+  const popupMenu = document.getElementById('popup_menu');
+  popupMenu.style.display = 'block';
+}
+
+function closeMenu() {
+  const popupMenu = document.getElementById('popup_menu');
+  popupMenu.style.display = 'none';
+}
+
+function createFeature(vehicle) {
+  const feature = `
+    <feature class="feature">
+      <img src="${vehicle.image}" alt="featured_image">
+      <info class="feature_info">
+        <h2 class="feature_title">${vehicle.name}</h2>
+        <p class="feature_subtitle">From ${vehicle.dealer} in ${vehicle.country}</p>
+        <hr>
+        <ul class="feature_specs">
+          <li>year: ${vehicle.year} </li>
+          <li>Engine: ${vehicle.engine} </li>
+          <li>hp: ${vehicle.horsepower} </li>
+        </ul>
+        <a onclick="showFeaturePopup(${vehicle.id})" href="#featured_content">More details</a>
+      </info>
+    </feature>`;
+  const parent = document.getElementById('featured_content');
+  parent.innerHTML += feature;
+}
+
+function listVehicles() {
+  const vehicle1 = {
+    id: 1,
+    name: 'AudiR8',
+    year: 2022,
+    engine: '5.2L V10',
+    horsepower: '562',
+    transmission: 'manual',
+    image: 'images/about_page/featured/audiR8_featured.png',
+    dealer: 'Extreme auto dealers',
+    country: 'Kenya',
+  };
+  const vehicle2 = {
+    id: 2,
+    name: 'Ford Mustang Roush',
+    year: 2018,
+    engine: '5.0L V8',
+    horsepower: '710',
+    transmission: 'manual',
+    image: 'images/about_page/featured/ford-mustang-roush.png',
+    dealer: 'Cranes auto',
+    country: 'Uganda',
+  };
+  const vehicle3 = {
+    id: 3,
+    name: 'Rolls-Royce Phantom',
+    year: 2003,
+    engine: '6.75L V12',
+    horsepower: '453',
+    transmission: 'automatic',
+    image: 'images/about_page/featured/rolls-royce-phanton.png',
+    dealer: 'Kigali auto dealers',
+    country: 'Rwanda',
+  };
+  const vehicle4 = {
+    id: 4,
+    name: 'Bentley Continental',
+    year: 2022,
+    engine: '4.0L V8',
+    horsepower: '542',
+    transmission: 'manual',
+    image: 'images/about_page/featured/bentley_continental.png',
+    dealer: 'Habesha Automotives',
+    country: 'Ethiopia',
+  };
+  return [vehicle1, vehicle2, vehicle3, vehicle4];
+}
```

Comment:
Same kindly make sure that your project sticks as much as possible to the template provided. The template in this section contains 6 cards however your project just contains 4. Kindly add 2 more cards. :eyes: :eyes: 
Your project
![image](https://user-images.githubusercontent.com/98363075/203805445-a5fda80a-b598-42fe-9fb6-beb67a0ef1fe.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203805533-c906a563-ebba-469a-9d0e-3be828891373.png)